src/foam/mlang/mlangJava.js

```diff
@@ -280,6 +305,10 @@ foam.CLASS({
       name: 'f',
       // TODO(adamvy): Is there a better option than all the Comparable casts?
       javaCode: 'return ((Comparable)getArg1().f(obj)).compareTo((Comparable)getArg2().f(obj)) == 0;'
+    },
+    {
+      name: 'toSQL',
+      javaCode: 'return getArg1().f(obj).toString() + "=" + getArg2().f(obj).toString();'
```

Comment:
Call toSQL() on the args, not toString(). toSQL() for ConstantExpr will have to escape HTML so that we don't get SQL injection errors.